### PR TITLE
Case study part 2 Ian Dessalet

### PR DESCRIPTION
**Pull request by idessalet-htc**

### CODE REVIEW

**Comment by mbMosman** [comment]
Ian, the new page HTML looks OK, but there shouldn't be an address there and the images/CSS updates are missing.

**Comment by mbMosman** [comment]
Did you mean to delete the activities HTML page?  It had some things wrong, but now it's totally gone.

File: activities.html [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+    <meta charset="uft-8">
+    <link rel="stylesheet" href="stylesheet.css">
+</head>
+
+<body>
+    <header>
+        <h1>Pacific Trails Resort</h1>
+    </header>
+    <nav>
+        <a href="index.html">Home</a>&nbsp;<a href="yurts.html">Yurts</a>&nbsp;<a href="activites.html">Activites</a>&nbsp;<a href="reservations.html">Reservations</a>
```
Comment (mbMosman) [inline activities.html]: You've got a typo here for the activities link in both the href and the text.  This is on all the pages and makes the links broken.

File: activities.html [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+        <h2><span class="resort">Actvites at Pacific Trails</span><h2>
+        <h3>Hiking</h3>
+        <p>pacific trails resort has 5 miles fo hiking trails and is adjacent to a state park go it alone or join one of our guided hikes.</p>
+        <h3>Kayaking</h3>
+        <p>ocean kayaks are avalable for guest use.</p>
+        <h3>Bird Watching</h3>
+        <p>While anytime is a good time for watching at pacific trails, we offer guided birdwtching trips at sunrise several times a week
+       
+        
+        <div>
```
Comment (mbMosman) [inline activities.html]: This address should not be on the activities page.  Make sure to compare your page with the image from the book to see if they look the same.

**Comment by idessalet-htc** [comment]
Ian Dessalet CS part 4

**Comment by mbMosman** [comment]
Make sure you are validating:  https://html5.validator.nu/?doc=https%3A%2F%2Fidessalet-htc.github.io%2Fcs-pacific-trails%2F

**Pull request closed without merging**